juju/juju
# [JUJU-3407] Update cmd to v3.0.7

Update the cmd package to v3.0.7 to support additional enhancements in the autogenerated docs for commands.

## Checklist


## QA steps

Generate the documentation for the commands and check that we have the `juju [options]` generated for all the commands.

```sh
mkdir /tmp/docs
_build/linux_amd64/bin/juju documentation --split --out=/tmp/docs --url="https://something.com"
```
Check that the `juju [options]` is displayed for the commands.
```sh
grep "juju [options]" /tmp/docs/*.md | wc -l
220
```
